### ISSUE TITLE
Unexpected WARNING: No miners available: disabled by Watchdog

### ISSUE THREAD

**Issue by earlofhemsley** [RainbowMiner/RainbowMiner#2924]
I was updating configuration to alter my mining pool selection, and then after a configuration update, was met with `WARNING: No miners available: disabled by Watchdog`. 

I'm relatively new to RainbowMiner, and rifling through the attached debug file didn't seem to reveal anything obvious other than perhaps I shouldn't be using 2Miners?

[debug_2024-12-27.zip](https://github.com/user-attachments/files/18258461/debug_2024-12-27.zip)

The watchdog is not enabled, which makes the warning particularly unhelpful. :-/ If there is any way to make this error message clearer as to what is actually happening, assuming you can make more sense of the debug file than I can?

Thank you for your help.

**Comment by earlofhemsley** [comment]
![image](https://github.com/user-attachments/assets/1cb14e0f-1016-4cbc-9c13-67c6da4de443)

**Comment by Motophan** [comment]
This is normal behavior, you need to go back a few verisons and turn off auto update or it wont work at all. 

https://github.com/RainbowMiner/RainbowMiner/issues/2925

**Comment by RainbowMiner** [comment]
That normally happens if the config file is corrupted. Did you edit the json files directly?

**Comment by earlofhemsley** [comment]
> That normally happens if the config file is corrupted. Did you edit the json files directly?

I did not. I went through the configuration setup in the terminal. As of this morning I was able to pull myself out of this by disabling settings relating to heat control and disabling 0-watt miners, among other things, so I'm back up and running.

I will probably start using source control in the config directories to make sure that I can revert config changes in the future if this pops up again, but for now I am good. Thank you for your attention!
